packages/algorand-js/test/mocks/stateful.ts

```diff
@@ -1,76 +1,58 @@
-import { AppLocalState } from "algosdk";
+import { AssetDef, AssetHolding } from "algosdk";
 
 import { stringToBytes } from "../../src/lib/parsing";
+import { AppLocalStateM, SSCAttributesM, StackElem } from "../../src/types";
+
+const byteToStr = (str: string): string => { return stringToBytes(str).toString(); };
```

Comment:
can we rename it to convertToKey or something else ?

Reply:
renaming to `convertToKey`